ui/components/app/transaction-decoding/transaction-decoding.component.js

```diff
@@ -86,8 +92,13 @@ export default function TransactionDecoding({ to = '', inputData: data = '' }) {
 
         // transform tx decoding arguments into tree data
         const params = transformTxDecoding(decoding?.arguments);
+
         setTx(params);
 
+        const dict = await parameterProcessing(params);
+
+        setProcessedValues(dict);
+
         setLoading(false);
       } catch (error) {
         setLoading(false);
```

Comment:
I know this isn't new to this PR but I think `setLoading(false);` can be removed from the `try` and `catch`, and instead be added with a `finally` block